components/dropdown/dropdown-opener-mixin.js

```diff
@@ -112,15 +130,18 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 		opener.removeAttribute('active');
 	}
 
-	__onKeyPress(e) {
-		if (e.keyCode !== 13) return;
+	__onKeyUp(e) {
 		if (this.noAutoOpen) return;
-		this.toggleOpen(true);
-	}
-
-	__onMouseUp() {
-		if (this.noAutoOpen) return;
-		this.toggleOpen(false);
+		switch (e.keyCode) {
+			case keyCodes.ENTER:
+			case keyCodes.SPACE:
```

Comment:
I also fixed an accessibility issue!

Reply:
Thanks Maya.  Instead of a switch, I'd be tempted to just use:
```
if (e.keyCode !== keyCodes.ENTER && e.keyCode !== keyCodes.SPACE) return;
this.toggleOpen(true);
...
```
I assume preventing default is for the Space key, which would normally scroll the page?  What's the reason doe stopping the propagation?